Add Pixhack v3 flight controller doc

hamishwillee opened the pull request:
This replaces #117

hamishwillee commented:
@cuavbin This moves the document to the right location. I also tidied the text to match the other flight controller documents.  Can you please check it looks OK to you. There is also a comment about the introduction.

hamishwillee commented on en/flight_controller/pixhack_v3.md:
```diff
@@ -0,0 +1,52 @@
+# Pixhack V3
+
+The CUAV *Pixhack V3* autopilot is a flexible autopilot intended primarily for manufacturers of commercial systems.  
```
@cuavbin Is this whole section correct? I ask because it looks a lot like the information that applies to https://docs.px4.io/en/flight_controller/pixhawk-2.html and it does not match the information you have in your documentation http://doc.cuav.net/PixHack/pixhack-v3.html.

- Can you confirm that this is based on **FMUv3** . If not, what FMU version is it?
- Can you confirm that this is based on the PH2? If not, what design is it closest to?
- Can you confirm that this has a separate carrier board. If so, then can you show an image of that?

If not, then this should have a quick summary of the key selling point of your boards. It could say things like
- Fast processor with failsafe processor
- Redundancy design
- Lots of expansion
- Built-in shock absorption system

Thanks!

cuavbin replied:
Hello, hamishwillee
Pixhack v3 really is based on PH2 (SOLO) hardware to develop
One of China's most popular flight controllers.
We have done a lot of optimization, such as damping system and thermostat system, as well as the interface layout optimization.
The documentation may be Chinese at this time, and we are about to switch it automatically according to the viewer's language. 
We also open the design files at our cuav GitHub
https://github.com/cuav/CUAV_Hardware/tree/master/Pixhack_V3

hamishwillee replied:
Thanks. So are there any carrier boards?

hamishwillee commented:
Thanks @cuavbin 
So I have updated this to 
- link to the specific open files for your board
- remove the clarification that docs are in Chinese. 
- Updated the introduction to tidy the text a little and clarify some of the additional features. 

I am still concerned about the carrier board question - can you confirm if this is still a design feature you're exploring?

hamishwillee commented on en/flight_controller/pixhack_v3.md:
```diff
@@ -0,0 +1,54 @@
+
+![Pixhack v3](../../assets/flight_controller/pixhack_v3/pixhack_v3_157_large_default.jpg)
+
+The controller is designed to be used with a domain-specific carrier board in order to reduce the wiring, improve reliability, and ease of assembly. For example, a carrier board for a commercial inspection vehicle might include connections for a companion computer, while a carrier board for a racer could includes ESCs form the frame of the vehicle.
```
@cuavbin Please check this whole section. Is line 11 correct? If so and you're happy with the rest we can probably merge.

@bkueng Any comments?

cuavbin replied:
Line 11 is not wrong
This description is very good, thank you

bkueng replied:
It's confusing to me as well. I don't see any carrier board, from the picture it looks like a Pixhawk, where you attach the individual cables.

bkueng commented on en/flight_controller/pixhack_v3.md:
```diff
@@ -0,0 +1,54 @@
+
+The board is a variant of the SOLO Pixhawk 2 (PH2) flight controller, which is in turn based on the [Pixhawk-project](https://pixhawk.org/) **FMUv3** open hardware design. It runs PX4 on the [NuttX](http://nuttx.org) OS, and is fully compatible with both PX4 or ArduPilot (APM) firmware.
+
+*Pixhack V3* has significant improvements with respect to the original design, including better interface layout and the addition of vibration damping and a thermostat system
```
There's a missing point at the end.

cuavbin replied:
This is a complete flight controller
The lead connectors are in front and behind

hamishwillee commented:
@cuavbin @bkueng I have removed the text about separate carrier boards - there is no information about them anywhere on your site, documentation, or images, so this will just be confusing for users. I am going to merge as is. If at some later point carrier boards appear we can create a new PR to add information.

Thanks for your help!
